CMakeLists.txt

```diff
@@ -112,6 +129,12 @@ if(EDLIB_ENABLE_INSTALL)
 		NAMESPACE edlib::)
 	
 	# Create target 'install' for installing libraries.
-	install(TARGETS edlib EXPORT ${targets_export_name} DESTINATION ${CMAKE_INSTALL_LIBDIR})
+	install(
```

Comment:
- Regarding SOVERSION, yes it should be incremented only for API/ABI breaking change, so major version if you follow semver. By the way, does meson build properly handle so/dylib names?

- indentation is not very consistent in edlib, there is a mix of tab and spaces.

Reply:
Ok sure, let's go with SOVERSION being the major version!
Indentation -> well in codebase I am pretty sure it is consistent and is all spaces. Maybe here in CMake somehow a tab got through, but we normally stick to spaces.

Reply:
Yes seems that there are tabs in `CMakeLists.txt` only (in `EDLIB_ENABLE_INSTALL` block specifically). I can replace those tabs in this PR if you want.

Reply:
Please do, and good job catching this! It must have happened in the recent PR that added that block.

Reply:
For pkg-config file, do you also want to always add major version as a suffix?

Currently in CMakeLists:

```cmake
configure_file(
    ${CMAKE_CURRENT_SOURCE_DIR}/edlib.pc.in
    ${CMAKE_CURRENT_BINARY_DIR}/edlib-1.pc
    @ONLY)

if(EDLIB_ENABLE_INSTALL)
  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/edlib-1.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
```

I would replace `edlib-1.pc` by `edlib-${PROJECT_VERSION_MAJOR}.pc`

What's the name of pkg-config file generated by meson?

Reply:
That makes a lot of sense, awesome!
I actually don't know hm, but I can add that to the checklist above as something we should check. I believe right now it generates some default pkg-config file, but it can be further configured.